packages/ui/src/components/List/ListContext.tsx

```diff
@@ -39,12 +40,14 @@ type ListProviderProps = {
   children: ReactNode
   autoCollapse: boolean
   selectable: boolean
+  expandCTA: boolean
```

Comment:
Im not a big fan of the wording for this variable, `expandButton` might be more in accordance to what we use among components